### DISCUSSION TITLE
Rebrand the app and/or put this fork in the Mac App Store?

### DISCUSSION THREAD

**Discussion by kethinov** [chucker/Mastonaut#85] [General]
New users to Mastodon looking for clients often find Mastonaut on lists of good Mastodon clients, but typically those lists point to the Mac App Store version which few people realize is not being maintained and fewer still are going to stumble on [this post letting people know about this fork](https://mastodon.social/@com/109367074091298209).

As far as I know there is no straightforward process to just take over the unmaintained Mac App Store version, so it seems the only option would be to rebrand the app with a new name and dock icon and submit it that way. (Hopefully Apple permits that...)

Even if you can't or don't want to submit the app to the Mac App Store (as I understand it, it's quite a hassle), rebranding would still be useful so public lists of good Mastodon clients can easily distinguish between the original app and this fork, e.g. similar to the distinction between MySQL and MariaDB and whatnot. This will make it easier for non-tech people to navigate the client ecosystem.

**Comment by chrisale** [comment]
This makes a lot of sense.Sent from my iPhoneOn Jan 3, 2023, at 22:37, Eric Newport ***@***.***> wrote:﻿
New users to Mastodon looking for clients often find Mastonaut on lists of good Mastodon clients, but typically those lists point to the Mac App Store version which few people realize is not being maintained and fewer still are going to stumble on this post letting people know about this fork.
As far as I know there is no straightforward process to just take over the unmaintained Mac App Store version, so it seems the only option would be to rebrand the app with a new name and dock icon and submit it that way. (Hopefully Apple permits that...)
Even if you can't or don't want to submit the app to the Mac App Store (as I understand it, it's quite a hassle), rebranding would still be useful so public lists of good Mastodon clients can easily distinguish between the original app and this fork, e.g. similar to the distinction between MySQL and MariaDB and whatnot. This will make it easier for non-tech people to navigate the client ecosystem.

**Comment by chucker** [comment]
Hi,

I understand the concern here. Indeed, I too have witnessed people concluding, "oh, I guess it isn't getting updated any more".

First, for reference: Bruno, the original maintainer, [cannot maintain it any longer.](https://web.archive.org/web/20220425150234/https://mastodon.technology/@brunoph/108138252725934908) (The thread is gone because that instance is gone.) It used to be a commercial app on the App Store. He made it free because he isn't working on it any longer, but making it free of course also means that the barrier to installing it is lower. So I'm guessing there are still people who discover+install it that way. But as he himself points out ("it has started to lag in features and functionality when compared to the Mastodon web client and other apps"), it'll start becoming less usable over time.

Now, I've forked it shortly after that announcement, more as a "what if" toy to play with, and also because I have a few things I'm interested in seeing in Masto clients. So far, I've done OK averaging a day or two per month of work into it. But I can't really commit to that in the long run, and even if I could, I feel like it's a bit too little. Issues on GitHub have been growing faster than I can resolve them.

I'm not too keen on either of these suggestions.

**Rebranding:** I don't want to take credit for something which Bruno has built 95% of. So I went with changing the about box so that it perpetually shows _his_ name and also the name of this project. I also feel like giving it a new name would make the "oh, actually, there _are_ updates for Mastonaut, just under a new name" kinds of replies even more complicated. Right now, people can search (e.g. for the `#Mastonaut` hashtag on Masto itself) and see that there's stuff going on.

I also don't have a great idea for a new name. It would also need a new icon and everything, and I don't have the time or expertise in giving it a good brand. Now, *if* someone has a great idea and wants to donate a design or offer it as a low fee, I might be up for that. (If so, we should have a separate discussion from this one.) But for now, I feel like "Mastonaut, but not quite the same one as on the App Store" is the lesser evil.

**App Store:** there are even more roadblocks here. I believe I'd have to negotiate with Bruno on having a version that isn't under the GPL, due to Apple's policies. And if I were to do that, I think a more logical path is to ask him to transfer the domain `mastonaut.app` to me, and also the App Store app (or to delist it), and/or update those to have them point to this project instead.

OK, sorry for the rambling. I guess the TL;DR is that I appreciate this discussion and do understand the ideas, but I feel I need to wait and see what the uptick is like. Maybe if one or two more people start contributing code, we can figure out a way to run this more professionally.

**Reply by lunasophia** [reply]
One way to address the "not enough tuits" problem is to convert the repository into an organization so that other people can merge PRs, do releases, and all the other project managery stuff. That means selecting people you trust to steward the project, of course, which has its own time costs that you could spend writing code. Trust me, I understand :) But I think that, long term, that would be the best choice for the project's future.

I'm sadly not really able to volunteer for that; my Swift knowledge is about four major versions out of date and my macOS programming knowledge is about 12 years out of date. But I'm sure others, like those who have open PRs, would delight in the opportunity!
